src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -95,7 +95,7 @@ const Row = ({
       <td className="key">{title}</td>
       <td className="value">
         <pre className="val " style={{backgroundColor: '#F0ECF3'}}>
-          <span className="val-string">{children}</span>
```

Comment:
@doralchan just a heads up, the removal of this classname will affect the font colour of the value field (from purple to black)

<img width="1038" alt="Screen Shot 2019-12-03 at 4 42 17 AM" src="https://user-images.githubusercontent.com/139499/70039095-53dd4600-1587-11ea-89d5-def2a6e848a3.png">

Previously: 

<img width="1028" alt="Screen Shot 2019-12-03 at 4 43 30 AM" src="https://user-images.githubusercontent.com/139499/70039175-7c654000-1587-11ea-9cd4-6ac2d004fb78.png">

Reply:
Yep

Reply:
I just realized this should be consistent with the other code blocks in general

Reply:
![Screen Shot 2019-12-03 at 9 31 11 AM](https://user-images.githubusercontent.com/4830259/70074629-edb8e900-15af-11ea-865a-47cf93000a4a.png)

✔️ Fixed